pkgs/by-name/mi/missidentify/package.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, stdenv
+, fetchurl
+, autoreconfHook
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "missidentify";
+  version = "1.0";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/project/missidentify/missidentify/missidentify-${finalAttrs.version}/missidentify-${finalAttrs.version}.tar.gz";
+    hash = "sha256-nnIRN8hpKM0IZCe0HUrrJGrxvBYKeBmdU168rlo8op0=";
+  };
+
+  patches = [
+    # define PATH_MAX variable to fix a FTBFS in Hurd.
+    (fetchurl {
```

Comment:
Recommend to use `fetchpatch` instead.

Reply:
For static patches it is recommended to use fetchurl instead of fetchpatch (that is for dynamic patches).

Reply:
Sorry, what is "static patch" and "dynamic patch"?

Reply:
@Aleksanaa https://github.com/NixOS/nixpkgs/pull/289535#discussion_r1579187054

Reply:
Oh I see, thanks for clarification